packages/host/tests/acceptance/operator-mode-test.ts

```diff
@@ -1115,6 +1119,58 @@ module('Acceptance | operator mode tests', function (hooks) {
     await waitFor('[data-test-save-idle]');
   });
 
+  test<TestContextWithSave>('non-card instance change made in monaco editor is auto-saved', async function (assert) {
+    assert.expect(1);
+    let operatorModeStateParam = stringify({
+      stacks: [[]],
+      submode: 'code',
+      codePath: `${testRealmURL}README.txt`,
+    })!;
+    await visit(
+      `/?operatorModeEnabled=true&operatorModeState=${encodeURIComponent(
+        operatorModeStateParam,
+      )}`,
+    );
+    await waitUntil(() => find('[data-test-editor]'));
+
+    this.onSave((content) => {
+      if (typeof content !== 'string') {
+        throw new Error('expected string save data');
+      }
+      assert.strictEqual(content, 'Hello Mars');
+    });
+
+    setMonacoContent('Hello Mars');
+
+    await waitFor('[data-test-save-idle]');
```

Comment:
Nice tests!

This is not a problem with anything in this PR, but seeing that we need this because Monaco operates outside Ember’s test waiters makes me wonder if we can eventually reflect Monaco state with Ember test waiters 🤔

Reply:
the monaco state changes are synchronous. this is actually waiting for the save that happens in response to the monaco state change (which uses a 5 second timeout).